kv/txn.go

```diff
@@ -74,7 +74,7 @@ func RunInNewTxn(store Storage, retryable bool, f func(txn Transaction) error) e
 }
 
 var (
-	// Max retry count in RunInNewTxn
+	// maxRetryCnt in RunInNewTxn
```

Comment:
maxRetryCnt represents maximum retry times in RunInNewTxn

Reply:
Add `.` at the end of a line

Reply:
@zhexuany @zimulala Done.